### ISSUE TITLE
Add support to previous branch shorthand

### ISSUE THREAD

**Issue by fpaguiar** [joseluisq/gitnow#35]
Hi,

Can I request an enhancement for the `move` comand? I usually go between current and previous branch by using the `-` shorthand. Would you consider adding support for it?

```shell
~/ (master) $ move staging
~/ (staging) $ move -
~/ (master) $ 
```

Thank you!

**Comment by joseluisq** [comment]
The idea is not bad but I think we can use keybindings for movement instead. At least for forward and backward navigation if user is under the `move` command.

So I suggest: <kbd>alt + →</kbd> (forward) and <kbd>alt + ←</kbd> (backward).
Remember that Fish default directory navigation is also using those keybindings but since these ones only apply for directories it won't interfere with `move` keybindings in that case.

What do you think?

**Comment by fpaguiar** [comment]
Thanks for the quick response!

Tbh I'm quite new to Fish, so I've never used keybindings before. It sounds really nice though! 👍 

As long as there is a way to move to previous branch without typing its name, I'll be glad with any solution.

**Comment by joseluisq** [comment]
Released on [2.11.0](https://github.com/joseluisq/gitnow/releases/tag/2.11.0)
